Documentation/CraftingNuGetPackages.md

```diff
@@ -0,0 +1,69 @@
+# Crafting a NuGet Package for MSB4U
+
+> MSBuildForUnity consumes .NET based packages without any modifications to the package. This documentation describes how C++, WinRT or packages with Unity-specific contents can easily be adapted to work with MSBuildForUnity.
+
+## Quickstart Summary
+
+Though the use of a [Build Targets or Props](https://docs.microsoft.com/en-us/nuget/create-packages/creating-a-package#include-msbuild-props-and-targets-in-a-package) file inside a NuGet package, we can leverage properties provided by MSBuildForUnity to appropriately select the contents to bring into the Unity project. The following snippet is taken from [Microsoft.Windows.MixedReality.DotNetWinRT](https://www.nuget.org/packages/Microsoft.Windows.MixedReality.DotNetWinRT/) NuGet package. It brings in the `Unity` folder packaged in the root of the NuGet package to the output folder inside Unity.
```

Comment:
nit: It brings in the `Unity` folder packaged in the root of the NuGet package to the output folder inside Unity.

This line can make sense if you are familiar with the fact that NuGet packages contain a file structure/content. If you really know nothing about NuGet packages, it may be worth state that the nuget package contains a Unity folder prior to saying that the Unity folder can be the only item added to a consumer's unity project. for example:


Microsoft.Windows.MixedReality.DotNetWinRT contains a Unity folder with Unity content. The below snippet shows how to reference the Microsoft.Windows.MixedReality.DotNetWinRT NuGet package and only include the Unity folder in the package consumer's Unity project.